language-server/src/main/scala/scala/meta/languageserver/SymbolIndexer.scala

```diff
@@ -43,26 +43,36 @@ class SymbolIndexer(
       } // not interesting for this service
     } yield pos -> denotation
 
-  def goToDefinition(
+  private def resolvedNameAt(
```

Comment:
Can we move this method to the bottom? I'm trying to keep the private utilities grouped together.

Reply:
sure, that makes sense :)